RELEASE_NOTES.md

```diff
@@ -28,17 +32,50 @@ information, see [SAT in CSM](operations/system_admin_toolkit/about_sat/SAT_in_C
 * Added support to upgrade CSM through IUF
 * Added support for iSCSI based boot content projection for `rootfs` and `PE` images
 
+### New hardware support
+
+* *IMPORTANT* Systems with NVIDIA CPUs and GPUs must see [Known issues](#known-issues)
+
+## Improvements
+
 ### Automation improvements
 
+* CSM can now be upgraded with IUF.
+
 ### Base platform component upgrades
 
-| Platform Component           | Version        |
-|------------------------------|----------------|
-| `Kubernetes`                 | 1.24.17        |
-| `istio`                      | 1.19.10        |
-| `kiali`                      | 1.75.0         |
-| `Strimzi Kafka`              | 0.41.0         |
-| `Kyverno`                    | 1.10.7         |
+| Platform Component           | Version |
+|------------------------------|---------|
+| Kubernetes                   | 1.24.17 |
+| `containerd`                 | 1.5.16  |
+| `istio`                      | 1.19.10 |
+| `kiali`                      | 1.75.0  |
+| `Strimzi Kafka`              | 0.41.0  |
+| `Kyverno`                    | 1.10.7  |
+| `keycloak`                   | 21.1.1  |
+| `bitnami-etcd` on `ncn-mxxx` | 3.5.0   |
```

Comment:
We don't use bitnami etcd on the master nodes, just plain ol' etcd.

bitnami is just the vendor of the helm chart we use to deploy etcd inside the cluster.